tests/integration/fixtures.py

```diff
@@ -855,6 +856,10 @@ def factory(**kwargs) -> str:
 )
 
 
+# define markers programmatically via MarkGenerator (could also be moved to pytest.ini)
+MarkGenerator._markers.add("aws_validated")
```

Comment:
We should probably put this into the "root" conftest.py for now :thinking:

Reply:
Good point, updated 👍